tests/unittests/OperatorTest.cpp

```diff
@@ -4426,6 +4426,51 @@ TEST_P(OperatorTest, Tanh) {
   }
 }
 
+static FunctionTensorPair
+createAndInitBasicExpTest(glow::PlaceholderBindings &bindings,
```

Comment:
can you merge this with createAndInitBasicTanhTest and make it more generic?

essentially it's all the same except F->createExp("Exp", input); call.

(reuse that for tanh and for exp).

Reply:
It's not just Exp, in fact TanH, Log, Sigmoid all can be combined, and strange it was never observed before. In future I'd rather have it addressed as a separate MR for test cleanup.